src/components/dictionaryConcepts/containers/CreateConcept.jsx

```diff
@@ -262,6 +264,7 @@ Concept
                 path={path}
                 state={this.state}
                 handleChange={this.handleChange}
+                disableButton={this.props.loading}
```

Comment:
@ozezzy, I think it is best to destructure all props and states;

Reply:
done